pinc/POFile.inc

```diff
@@ -126,76 +128,85 @@ class POFile
 
     public function create_template($basedir, $pot_filename)
     {
-        $cmd = join(" ", [
-            // change into the base directory so the filenames in the POT
-            // don't have it prefixed to them
-            "cd " . escapeshellarg($basedir),
-            "&&",
+        // create the new template in a temporary file in case xtext fails
```

Comment:
Perhaps I'm misunderstanding what I'm supposed to be testing, so I figured I'd better ask. Comparing against the same action on TEST, I regenerated the POT. Two things I noticed:
* It took a  _**lot**_ longer for the template to be regenerated in the [remove-remaining-shell/](https://www.pgdp.org/~cpeel/c.branch/remove-remaining-shell/) sandbox.
* The "last modified" timestamp was exactly the same. I just tried regenerating it again in the sandbox, and got the same results (It's giving me the message that xtext ran successfully).
* I switched back to TEST, and got an updated timestamp; switched back to the sandbox, and the timestamp matched TEST, once again. Same extra-long wait time for the template to be regenerated, same message saying that the file was successfully generated.

I only downloaded and compared the POT files for the first time around -- as expected, they matched.

I still need to do other testing, but it's time for me to stop for the night, and I wanted to go ahead and mention this.

Reply:
This is my mistake. I neglected to update the sandbox after fixing the logic error @mrducky4 found in the PR. I've updated it now. `POFile.inc` was the only file changed in that update.